server/study_groups/methods.js

```diff
@@ -454,3 +454,59 @@ Meteor.methods({
     return true;
   }
 });
+
+Meteor.methods({
+  transferStudyGroup: function(data) {
+    check(data, {
+      studyGroupId: String,
+      newOwnerId: String,
+      studyGroupTitle: String,
+      studyGroupSlug: String
+    });
+
+    if (!this.userId) {
+      throw new Meteor.Error('Members.methods.editStatus.not-logged-in', 'Must be logged in to transfer study group.');
+    }
+
+    const actor = Meteor.user()
+    if (!actor || !Roles.userIsInRole(actor, ['owner','admin', 'moderator', 'member'], data.studyGroupId )) {
+      throw new Meteor.Error(403, "Access denied");
+    }
+
```

Comment:
Here, we should also check if the `newOwnerId` is belong to any of the member or not.

`if(!Roles.userIsInRole(data.newOwnerId, ['admin', 'moderator', 'member'], data.studyGroupId ){`